README.md

```diff
@@ -56,6 +56,7 @@ All resources involving the use of, building application on, or developing for,
 
 * [Ethermint](http://github.com/tendermint/ethermint); Ethereum on Tendermint
 * [Cosmos SDK](http://github.com/cosmos/cosmos-sdk); a cryptocurrency application framework
+* [Passchain](http://github.com/trusch/passchain); a secret sharing system for teams
```

Comment:
I feel like we should have a "community built" section here and list a bunch of them ...